tests/driver_sx127x/main.c

```diff
@@ -302,10 +305,10 @@ static void _event_cb(netdev_t *dev, netdev_event_t event)
             case NETDEV_EVENT_RX_COMPLETE:
```

Comment:
(we could tag the pkt->type to a LoRaWAN packet, and automatically pass the packet to application with [this line](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/link_layer/netdev/gnrc_netdev.c#L71) ). Any comments? :)